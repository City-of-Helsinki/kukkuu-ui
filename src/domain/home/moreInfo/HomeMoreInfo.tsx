import React from 'react';
import { useTranslation } from 'react-i18next';

import styles from './homeMoreInfo.module.scss';
import MoreInfoLinkList from './MoreInfoLinkList';
import { moreInfoLinks } from './constants/MoreInfoConstants';

const HomeMoreInfo = () => {
  const { t } = useTranslation();

  // <Partners size="big" partners={mainPartnerList} />
  return (
    <section className={styles.wrapper}>
      <div className={styles.innerwrapper}>
        <h2>{t('home.moreInfo.heading.text')}</h2>
        <p>{t('home.moreInfo.description.text')}</p>
        <MoreInfoLinkList links={moreInfoLinks} />
      </div>
    </section>
  );
};

export default HomeMoreInfo;
