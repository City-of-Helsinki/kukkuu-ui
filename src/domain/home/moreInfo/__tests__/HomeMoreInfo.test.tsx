import { render } from '../../../../common/test/testingLibraryUtils';
import HomeMoreInfo from '../HomeMoreInfo';

it('renders snapshot correctly', () => {
  const { container } = render(<HomeMoreInfo />);
  expect(container).toMatchSnapshot();
});
