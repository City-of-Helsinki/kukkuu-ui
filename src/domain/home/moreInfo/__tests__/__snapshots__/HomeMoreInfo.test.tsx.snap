// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`renders snapshot correctly 1`] = `
<div>
  <section
    class="_wrapper_557708"
  >
    <div
      class="_innerwrapper_557708"
    >
      <h2>
        Lisätietoa palvelusta eri kielillä
      </h2>
      <p>
        Lataa lisätietoa Kultuurin kummilapset -palvelusta omalla kielelläsi.
      </p>
      <div
        class="_link_0c660e"
      >
        <a
          href="undefined/brochures/fr.pdf"
        >
          <span
            lang="fr"
          >
            Français
          </span>
          <span>
             
          </span>
          <span
            class="_localizedLanguage_0c660e"
            lang="fi"
          >
            (
            ranska
            )
          </span>
        </a>
        <a
          href="undefined/brochures/ru.pdf"
        >
          <span
            lang="ru"
          >
            Русский
          </span>
          <span>
             
          </span>
          <span
            class="_localizedLanguage_0c660e"
            lang="fi"
          >
            (
            venäjä
            )
          </span>
        </a>
        <a
          href="undefined/brochures/es.pdf"
        >
          <span
            lang="es"
          >
            Español
          </span>
          <span>
             
          </span>
          <span
            class="_localizedLanguage_0c660e"
            lang="fi"
          >
            (
            espanja
            )
          </span>
        </a>
        <a
          href="undefined/brochures/zh.pdf"
        >
          <span
            lang="zh"
          >
            中文
          </span>
          <span>
             
          </span>
          <span
            class="_localizedLanguage_0c660e"
            lang="fi"
          >
            (
            kiina
            )
          </span>
        </a>
        <a
          href="undefined/brochures/npi.pdf"
        >
          <span
            lang="npi"
          >
            नेपाली गण
          </span>
          <span>
             
          </span>
          <span
            class="_localizedLanguage_0c660e"
            lang="fi"
          >
            (
            nepali
            )
          </span>
        </a>
        <a
          href="undefined/brochures/som.pdf"
        >
          <span
            lang="som"
          >
            Somali
          </span>
          <span>
             
          </span>
          <span
            class="_localizedLanguage_0c660e"
            lang="fi"
          >
            (
            somali
            )
          </span>
        </a>
        <a
          href="undefined/brochures/fas.pdf"
        >
          <span
            lang="fas"
          >
            زبان فارسی
          </span>
          <span>
             
          </span>
          <span
            class="_localizedLanguage_0c660e"
            lang="fi"
          >
            (
            persia
            )
          </span>
        </a>
        <a
          href="undefined/brochures/et.pdf"
        >
          <span
            lang="et"
          >
            Eesti
          </span>
          <span>
             
          </span>
          <span
            class="_localizedLanguage_0c660e"
            lang="fi"
          >
            (
            viro
            )
          </span>
        </a>
        <a
          href="undefined/brochures/tur.pdf"
        >
          <span
            lang="tur"
          >
            Türkçe
          </span>
          <span>
             
          </span>
          <span
            class="_localizedLanguage_0c660e"
            lang="fi"
          >
            (
            turkki
            )
          </span>
        </a>
        <a
          href="undefined/brochures/ara.pdf"
        >
          <span
            lang="ar"
          >
            اَلْعَرَبِيَّةُ
          </span>
          <span>
             
          </span>
          <span
            class="_localizedLanguage_0c660e"
            lang="fi"
          >
            (
            arabia
            )
          </span>
        </a>
        <a
          href="undefined/brochures/ckb.pdf"
        >
          <span
            lang="ckb"
          >
            کوردی
          </span>
          <span>
             
          </span>
          <span
            class="_localizedLanguage_0c660e"
            lang="fi"
          >
            (
            kurdi
            )
          </span>
        </a>
        <a
          href="undefined/brochures/vi.pdf"
        >
          <span
            lang="vi"
          >
            Tiếng Việt
          </span>
          <span>
             
          </span>
          <span
            class="_localizedLanguage_0c660e"
            lang="fi"
          >
            (
            vietnam
            )
          </span>
        </a>
        <a
          href="undefined/brochures/en.pdf"
        >
          <span
            lang="en"
          >
            English
          </span>
          <span>
             
          </span>
          <span
            class="_localizedLanguage_0c660e"
            lang="fi"
          >
            (
            englanti
            )
          </span>
        </a>
        <a
          href="undefined/brochures/sv.pdf"
        >
          <span
            lang="sv"
          >
            Svenska
          </span>
          <span>
             
          </span>
          <span
            class="_localizedLanguage_0c660e"
            lang="fi"
          >
            (
            ruotsi
            )
          </span>
        </a>
        <a
          href="undefined/brochures/fi.pdf"
        >
          <span
            lang="fi"
          >
            Suomi
          </span>
          <span>
             
          </span>
          <span
            class="_localizedLanguage_0c660e"
            lang="fi"
          >
            (
            suomi
            )
          </span>
        </a>
      </div>
    </div>
  </section>
</div>
`;
